guacamole/src/main/webapp/app/index/styles/ui.css

```diff
@@ -150,6 +150,12 @@ div.section {
     padding: 0.1em;
 }
 
+.list-item .caption:after {
```

Comment:
Applies clearfix in order to make the container tag be the full height of all floated and non-floated elements within it.

Example: https://css-tricks.com/snippets/css/clear-fix/